src/vs/workbench/contrib/search/browser/searchActions.ts

```diff
@@ -595,7 +595,7 @@ export class RemoveAction extends AbstractSearchAndReplaceAction {
 			this.getElementToFocusAfterRemoved(this.viewer, <any>currentBottomFocusElement) :
 			null;
 
-		if (nextFocusElement) {
+		if (nextFocusElement && !opInfo.elements.includes(nextFocusElement)) {
```

Comment:
If `nextFocusElement` is in the set that is being removed, it seems like something went wrong in computing `nextFocusElement`. Maybe the problem is that `this.getElementToFocusAfterRemoved` is only looking at the one last element, and not the full set that is being removed?

It seems like whatever is going on in the line above, it should give us a valid element.